### ISSUE TITLE
How to tell someone to distance themselves in the city?

### ISSUE THREAD

**Issue by scripting** [scripting/Scripting-News#167]
A friend from NYC tells a story of a bagel and coffee and a NYC bakery.

**Comment by pwiner** [comment]
In that specific situation, I would probably keep my distance and not say anything.

If I'm walking outside and someone is approaching too closely, I extend my arms like a baseball umpire indicating 'safe'.  If they don't respond, I move my arms up and down to get their attention.  If that still doesn't work, I use my full voice, not shouting and respectfully say, "distance please".

If none of that works, I jump out of the way.

**Comment by jfreeman** [comment]
I agree with Dave here.. you can only control for yourself.. if there's room, keep your distance.  If there's not, go outside and wait until there's room.  If it takes a long time maybe poke your head in and say with a smile that you'd like to come in and do business but need to feel a little safer than the situation currently allows..

**Comment by scripting** [comment]
My brother @pwiner pretty much has the right idea of course. Not much you can really do but keep your distance, and shrug it off. And maybe don't go back to that bakery, no matter how good their bagel is. 

But here's my fantasy answer which comes from having ridden a bike in urban areas for many years. I dream of carrying a can of red spray paint with me, and when someone comes too close with their car and I have to slam on the brakes to avoid getting run over, I whip out the paint and put a red stripe on their fender or bumper. Then he would have to explain to his friends and family how he got the red stripe. And shitty drivers would end up with LOTS of them. 

What do you think??

**Comment by epaull** [comment]
"red spray paint" -- genius, I love it. 👍 Would love to do that, but too many Americans have guns :(. Maybe I use a stick.

**Comment by drewkime** [comment]
If cyclists and/or pedestrians started doing this, I'm convinced certain drivers would start trying for the highest score.

**Comment by iolaire** [comment]
Even with a mask it sounds like a situation that should be entirely avoided by the friend.  You should not risk your health or life for a single coffee and bagel.  My recommendation would be to use this as a learning opportunity and figure out how to stock up their apartment so these types of trips are not needed.  

For example wait until tomorrow and go buy a weeks worth of bagels early in the morning when the shop opens, and figure out how to insure they can make their own coffee.  Maybe they can not toast their own bagel, but deal with it.

**Comment by GeoffMeredith** [comment]
Having a 6ft pole to poke people might help for keeping maintaining distance.  An overcrowded local walking park, instead of closing, sent out park workers with hula hoops, pool noodles, and hockey sticks to help demonstrate safe distancing.
I've slapped or tapped windows or fenders of cars hundreds of times when they've come too close.  I think that people mostly don't realize what they are doing and a little reminder can help.  I live in Canada and would be less likely to do this in the US.
